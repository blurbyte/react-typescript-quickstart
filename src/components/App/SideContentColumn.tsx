import styled from '@styles/styledComponents';

const SideContentColumn = styled.section`
  grid-column: -2 / -1;
`;

export default SideContentColumn;
