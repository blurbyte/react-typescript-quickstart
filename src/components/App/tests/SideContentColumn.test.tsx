import * as React from 'react';
import { shallow } from 'enzyme';

import SideContentColumn from '../SideContentColumn';

describe('SideContentColumn', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<SideContentColumn />);
    expect(wrapper).toMatchSnapshot();
  });
});
