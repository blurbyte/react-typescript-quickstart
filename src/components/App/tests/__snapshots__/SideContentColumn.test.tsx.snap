// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideContentColumn renders correctly 1`] = `
.c0 {
  grid-column: -2 / -1;
}

@media (max-width:62.5em) {
  .c0 {
    grid-column: 1 / -1;
  }
}

<section
  className="c0"
/>
`;
